tornado-runtime/src/main/java/uk/ac/manchester/tornado/runtime/analyzer/TaskUtils.java

```diff
@@ -312,15 +312,14 @@ private static DomainTree buildDomainTree(int[] dims) {
     }
 
     public static PrebuiltTask createTask(ScheduleMetaData meta, PrebuiltTaskPackage taskPackage) {
-        DomainTree domain = buildDomainTree(taskPackage.getDimensions());
+        //DomainTree domain = buildDomainTree(taskPackage.getDimensions());
+        DomainTree domain = null;
```

Comment:
Remove the domain. I think it is not used any more.

Reply:
Do we need the `buildDomainTree` method?

Reply:
Yes, we can remove the `domain` at this point.